ocaml/parsing/lexer.mll

```diff
@@ -51,6 +51,7 @@ let keyword_table =
     "else", ELSE;
     "end", END;
     "exception", EXCEPTION;
+    "exclave_", EXCLAVE;
```

Comment:
This change means you'll need a Merlin update, but the scripts should handle that correctly now.